modules/tinymce/src/core/test/ts/browser/html/SaxParserTest.ts

```diff
@@ -284,6 +284,20 @@ describe('browser.tinymce.core.html.SaxParserTest', () => {
     parser.parse('<p title="gre>gererg"/>');
     assert.equal(writer.getContent(), '<p title="gre&gt;gererg"></p>', 'Parse elements with quoted > characters.');
     assert.deepEqual(counter.counts, { start: 1, end: 1 }, 'Parse elements with quoted > characters (counts).');
+
+    counter = createCounter(writer);
+    parser = SaxParser(counter, schema);
+    writer.reset();
+    parser.parse('<iframe id="someid" src="test" name="my_iframe"></iframe>');
```

Comment:
```suggestion
    const content = '<iframe id="someid" src="test" name="my_iframe"></iframe>';
    parser.parse(content);
    assert.equal(writer.getContent(), content, 'Parse iframe elements with name and id attributes.');
```
Just less code

Reply:
I would also add one or more attributes

Reply:
Theres dozens of cases in each test in this file, so I don't want to declare new const variables for each case.